src/applications/ivc-champva/10-10D/pages/ApplicantSponsorMarriageDetailsPage.jsx

```diff
@@ -99,10 +102,25 @@ export const marriageDatesSchema = {
   uiSchema: {
     applicants: {
       items: {
-        'ui:options': { viewField: ApplicantField },
-        ...titleUI('Applicant date of marriage to sponsor'),
+        'ui:options': {
+          viewField: ApplicantField,
+          updateSchema: formData => {
+            return {
+              title: context =>
+                titleUI(
+                  `${applicantWording(
+                    formData,
+                    context,
+                    true,
+                    true,
+                  )} marriage dates`,
+                  '',
+                )['ui:title'],
+            };
+          },
+        },
```

Comment:
Using `updateSchema` to get applicant-specific wording in the title of this page. When inside a list and look like this, have to use this method to update title wording based on state of the applicants list.